test_cases/arc4_types/dynamic_bytes.py

```diff
@@ -25,14 +25,19 @@ def approval_program(self) -> bool:
         for uint8_item in dynamic_bytes2:
             total += uint8_item.native
 
+        for uint8_item in dynamic_bytes2:
```

Comment:
Assertion messages says `dynamic_bytes3` but this iterates `dynamic_bytes2`

Reply:
Good spot, thanks